UndertaleModLib/Compiler/Lexer.cs

```diff
@@ -844,7 +843,6 @@ public enum TokenKind
                     KeywordExit,
                     KeywordBreak,
                     KeywordContinue,
-                    KeywordStruct, // Apparently this exists
```

Comment:
can we git blame who added this and figure out why it was added in the first place? feels very unlikely for this to get added when it never existed.

Reply:
I have a vague recollection that it could have been me, back when initially writing the GML compiler. I think it was because it was in some keyword list I was referencing somewhere, but it's been years so I really don't remember. What I *do* know is that there's no actual usage of this keyword anywhere, so it should be safe to remove.

Reply:
cool